cmd/vcert/cmdCloudProviders.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
Change the name of this file. This is provisioning, not cloudProviders. it should be `cmdProvisioning.go`

Reply:
resolved by latest changes